diff --git a/node_modules/@threlte/extras/dist/components/Environment/GroundProjectedEnv.svelte b/node_modules/@threlte/extras/dist/components/Environment/GroundProjectedEnv.svelte
index 3b31d7d..d5d805f 100644
--- a/node_modules/@threlte/extras/dist/components/Environment/GroundProjectedEnv.svelte
+++ b/node_modules/@threlte/extras/dist/components/Environment/GroundProjectedEnv.svelte
@@ -1,6 +1,6 @@
 <script>import { useThrelte } from '@threlte/core';
 import { onDestroy } from 'svelte';
-import { GroundProjectedEnv } from 'three/examples/jsm/objects/GroundProjectedEnv';
+import { GroundProjectedSkybox } from 'three/examples/jsm/objects/GroundProjectedSkybox';
 export let groundProjection;
 export let currentEnvMap;
 const { scene, invalidate } = useThrelte();
@@ -26,7 +26,7 @@ const toggleGroundEnv = (groundEnv, groundEnvProps, envMap) => {
     if (groundEnv && previousEnvMap != envMap)
         removeGroundEnv();
     if ((!groundEnv || previousEnvMap != envMap) && groundEnvProps && envMap) {
-        currentGroundEnv = new GroundProjectedEnv(envMap);
+        currentGroundEnv = new GroundProjectedSkybox(envMap);
         scene.add(currentGroundEnv);
         previousEnvMap = envMap;
         applyGroundParams();
